client/components/password-change/index.css

```diff
@@ -0,0 +1,49 @@
+.owisp-passchange-container {
+  width: 30vw;
```

Comment:
Please avoid using vh and vw for now. See styles of any other component and follow the same structure and styles for the password change component.